allure-attachments/src/main/java/io/qameta/allure/attachment/http/HttpRequestAttachment.java

```diff
@@ -226,6 +249,7 @@ private String getCurl() {
             builder.append(" '").append(url).append('\'');
             headers.forEach((key, value) -> appendHeader(builder, key, value));
             cookies.forEach((key, value) -> appendCookie(builder, key, value));
+            formParams.forEach((key, value) -> appendFormParams(builder, key, value));
```

Comment:
Possible NPE in case of using the old constructor. BTW, you can provide null values in constructors for all the fields.

Reply:
What should i do for this comment?
Create test for null values?